masterlist.yaml

```diff
@@ -712,3 +721,67 @@ plugins:
         <<: *dirtyPlugin
         itm: 4
         udr: 1
+  - name: 'AyleidCitadel.esp'
```

Comment:
It would be nice to have the `url` included for new entries as well, allowing other/future editors to easily look up the plugin. :) (Also, I've been talking a bit with [modwat.ch](https://modwat.ch/) about them using the LOOT masterlists to add links to plugins in their lists, which would also need the `url` attributes being set.)